client/src/main/java/com/tigrisdata/db/client/service/StandardTigrisCollection.java

```diff
@@ -151,6 +152,37 @@ public InsertResponse insert(T document) throws TigrisDBException {
     return insert(Collections.singletonList(document));
   }
 
+  @Override
+  public ReplaceResponse insertOrReplace(
```

Comment:
probably change ReplaceResponse and ReplaceRequestOptions as well.

Reply:
totally missed it. updated code.